clang/lib/DPCT/SaveNewFiles.cpp

```diff
@@ -30,7 +31,9 @@
 
 #include "Utility.h"
 #include "llvm/Support/raw_os_ostream.h"
+#include "llvm/Support/raw_ostream.h"
```

Comment:
Can we merge these LLVM header before L22? and seems this include order is not looks well.

Reply:
Fallback all the changes in this file.